src/classes/ALLO_AllocationsUtil_TEST.cls

```diff
@@ -420,6 +419,138 @@ public class ALLO_AllocationsUtil_TEST {
 
     }
 
+    /*******************************************************************************************************
```

Comment:
Since these are not actually directly AllocationsUtil methods, can you move them down after Helper Methods - Error Tests, and add a comment block called Helper Methods - Callable Support?